scene/3d/gpu_particles_3d.h

```diff
@@ -74,7 +75,7 @@ class GPUParticles3D : public GeometryInstance3D {
 	bool fractional_delta;
 	bool interpolate = true;
 	NodePath sub_emitter;
-	float collision_base_size;
+	float collision_base_size = 1.0;
```

Comment:
The constructor calls `set_collision_base_size(0.01)`, so this seems redundant / confusing that it's a different value.